### PR TITLE
Delivery Partnership Playbook > Partnership Playbook

### PR DESCRIPTION
**Pull request by wslack**
Per discussions with biz units

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -46,6 +44,8 @@ navigation:
   url: /lean-product-design
 - text: Open Source Guide
   url: /open-source-guide
+ - text: "Partnership Playbook"
```
Comment (ccostino) [inline _config.yml]: Hey @wslack, do you mind removing the extra space in the beginning here just to keep formatting nice in the file?  I'll merge afterward. :-)  Thanks!

File: _config.yml [hunk 2]
```diff
@@ -46,6 +44,8 @@ navigation:
...
+  url: /partnership-playbook
```
Comment (wslack) [inline _config.yml:48]: Of course! Will do in five min.

**Comment by wslack** [comment]
@ccostino ready now!

**Comment by ccostino** [comment]
Thanks @wslack!

**Pull request merged**